pkgs/top-level/python-packages.nix

```diff
@@ -31653,6 +31653,28 @@ EOF
     };
   };
 
+  pytoml = buildPythonPackage rec {
+    name = "pytoml-${version}";
+    version = "0.1.11";
+
+    checkPhase = "${python.interpreter} test/test.py";
+
+    # fetchgit used to ensure test submodule is available
+    src = pkgs.fetchgit {
```

Comment:
```fetchFromGitHub``` retrieves the tarball. We need a proper git checkout in order to get the submodule.